AnnoDesigner/ViewModels/MainViewModel.cs

```diff
@@ -917,6 +920,19 @@ public bool CanvasShowHarborBlockedArea
             }
         }
 
+        public bool CanvasShowPanorama
+        {
+            get { return _canvasShowPanorama; }
+            set
+            {
+                UpdateProperty(ref _canvasShowPanorama, value);
```

Comment:
Should we be doing this inside the null check? Surely this means that we could fire the event that the property has been updated, without updating the property?